src/utils/speech_to_text_api.py

```diff
@@ -0,0 +1,38 @@
+import os
+import io
+
+from google.cloud import speech
+
+class SpeechToTextAPI:
+
+        def __init__(self):
+            pass
+
+        @staticmethod
+        def resquestTranscription(serviceAccountPath, audioDataPath, encoding, sampleRate, languageCode, model, enhanced):
+            # Instantiates a client.
+            client = speech.SpeechClient.from_service_account_json(serviceAccountPath)
+
+            # The name of the audio data to transcribe.
+            file_name = os.path.join(audioDataPath)
+
+            # Loads the audio data into memory.
+            with io.open(file_name, 'rb') as audio_file:
```

Comment:
We should add a comment to explain what is the meaning of 'rb'. It would be nice